modules-js/react-fleet/src/form-elements/UploadPhoto.stories.tsx

```diff
@@ -40,6 +40,23 @@ storiesOf('Form Elements/Upload photo', module)
         handleRemove={() => {}}
       />
     </div>
+  ))
+  .add('displaying image File from data store', () => (
```

Comment:
This test is incomplete, because I was trying to use an actual photograph and was going to try doing so by pulling the `File` from the DOM, and then inserting it... would love to talk about this!